web_src/js/components/RepoActionView.vue

```diff
@@ -495,14 +495,22 @@ export default defineComponent({
             {{ locale.artifactsTitle }}
           </div>
           <ul class="job-artifacts-list">
-            <li class="job-artifacts-item" v-for="artifact in artifacts" :key="artifact.name">
-              <a class="job-artifacts-link" target="_blank" :href="run.link+'/artifacts/'+artifact.name">
-                <SvgIcon name="octicon-file" class="ui text black job-artifacts-icon"/>{{ artifact.name }}
-              </a>
-              <a v-if="run.canDeleteArtifact" @click="deleteArtifact(artifact.name)" class="job-artifacts-delete">
-                <SvgIcon name="octicon-trash" class="ui text black job-artifacts-icon"/>
-              </a>
-            </li>
+            <template v-for="artifact in artifacts" :key="artifact.name">
+              <li class="job-artifacts-item" v-if="artifact.status !== 'expired'">
+                <a class="job-artifacts-link" target="_blank" :href="run.link+'/artifacts/'+artifact.name">
+                  <SvgIcon name="octicon-file" class="ui text black job-artifacts-icon"/>{{ artifact.name }}
+                </a>
+                <a v-if="run.canDeleteArtifact" @click="deleteArtifact(artifact.name)" class="job-artifacts-delete">
+                  <SvgIcon name="octicon-trash" class="ui text black job-artifacts-icon"/>
+                </a>
+              </li>
+              <li class="job-artifacts-item" v-else>
+                <span class="ui text light grey">
+                  <SvgIcon name="octicon-file" class="ui text light grey job-artifacts-icon"/>{{ artifact.name }}
+                  <span class="ui text label light grey tw-ml-1">{{ locale.artifactExpired }}</span>
+                </span>
```

Comment:
I prefer that the expired label is aligned like the trash icon. e.g. http://localhost:3000/devtest/repo-action-view/0/0 (RUN_MODE = dev in app.ini)

![image](https://github.com/user-attachments/assets/86bb5e6c-90ae-4036-9a78-a71da5a681fa)
<img width="535" alt="image" src="https://github.com/user-attachments/assets/3a186c10-aed3-4da1-aef9-d1ed9dc88257" />

```suggestion
                <span class="ui text light grey">
                  <SvgIcon name="octicon-file" class="ui text light grey job-artifacts-icon"/>{{ artifact.name }}
                </span>
                <span class="ui text label light grey">{{ locale.artifactExpired }}</span>
```

**EDIT** paste the snipped, before it was no change

Reply:
good point, I guess your current style would wrap the label